slack.ts

```diff
@@ -0,0 +1,54 @@
+/** Error20:
+    - line 21: don't know how to declare call signature in tsc*/
+
+import * as querystring from "querystring";
+import fetch from "node-fetch";
+
+const tkn = process.env.SLACK_TOKEN;
+const domain = process.env.DOMAIN;
+
+async function sendMessage(message : string, config : string) : Promise<void> {
+    const msg = {
+        token: tkn,
+        channel: config,
+        as_user: false,
+        username: "HackGT",
+        text: message
+    };
+
+    const qs = querystring.stringify(msg);
+
+    const res = await
+        fetch(`https://${domain}.slack.com/api/chat.postMessage?${qs}`);
+    const json = await res.json();
+    console.log(json);
+}
+
+async function check(config : any): Promise<string> {
+    const res = await
```

Comment:
You should first check that `config` is the type that you want.
E.g. check that its a `string`, or a `{ channels: string[] }` type if your doing the multiple channels thing.